### ISSUE TITLE
[Crash] System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable)

### ISSUE THREAD

**Issue by gidsi** [sn4k3/UVtools#968]
### System

```Text
UVtools v5.0.2 X64
Operative system: Arch Linux X64
Processor: Intel(R) Core(TM) i5-4200U CPU @ 1.60GHz
Processor cores: 4
Memory RAM: 2.41 / 7.64 GB
Runtime: linux-x64
Framework: .NET 9.0.0
AvaloniaUI: 11.2.3
OpenCV: 4.9.0

Path:       /tmp/.mount_uvtoolS6oaJc/usr/bin/
Executable: /tmp/.mount_uvtoolS6oaJc/usr/bin/UVtools
```

### Printer and Slicer

- Printer: 
- Slicer: 

### Description of the bug

```
System.AggregateException: A Task's exception(s) were not observed either by Waiting on the Task or accessing its Exception property. As a result, the unobserved exception was rethrown by the finalizer thread. (org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable)
 ---> Tmds.DBus.Protocol.DBusException: org.freedesktop.DBus.Error.ServiceUnknown: The name is not activatable
   at Tmds.DBus.Protocol.DBusConnection.MyValueTaskSource`1.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)
   at Tmds.DBus.Protocol.DBusConnection.CallMethodAsync(MessageBuffer message)
   at Tmds.DBus.Protocol.Connection.CallMethodAsync(MessageBuffer message)
   --- End of inner exception stack trace ---
Category: Task
File: UVtools_demo_file.sl1s  [Version: 0] [Class: SL1File]

Machine date time: 21/12/2024 20:09:58
    UTC date time: 21/12/2024 19:09:58
```

### How to reproduce

Open settings and hover the ram limit

**Comment by sn4k3** [comment]
Duplicate of: https://github.com/sn4k3/UVtools/issues/964
